doc/Makefile

```
@@ -36,6 +37,15 @@ html: api
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
 
+html-noplot: api
+	@echo
+	@echo "Building HTML files without example plots."
+	@echo
+	# Set PYGMT_USE_EXTERNAL_DISPLAY to "false" to disable external display
+	PYGMT_USE_EXTERNAL_DISPLAY="false" $(SPHINXBUILD) -D plot_gallery=0 -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
```

Setting `PYGMT_USE_EXTERNAL_DISPLAY` to `false` seems unnecessary because no examples will be executed. Removing it works well for me locally, but please double check it.
```suggestion
	$(SPHINXBUILD) -D plot_gallery=0 -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
```

---

Yes, this also works well for me with your suggestion.